src/Explorer/SplashScreen/SplashScreenComponent.tsx

```diff
@@ -45,7 +45,7 @@ export class SplashScreenComponent extends React.Component<SplashScreenComponent
                 tabIndex={0}
                 role="button"
               >
-                <img src={item.iconSrc} alt={item.title} />
+                <img src={item.iconSrc} alt="" />
```

Comment:
Can you explain the purpose of the change? I thought this was required for those who can't see the image.

Reply:
Screen reader is reading this twice, once for the image alt and once for the button content. Since the image is acting as a button icon, we should remove the alt text.